client/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "birdseed_client"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+nostr-sdk = { version = "0.32.0", features = ["nip04"] }
```

Comment:
In case nip59 is suitable for our usecase, we could try to switch back to private_direct_message() again later for improved privacy.

Reply:
Yes, NIP-04 , must be replaced through the newest DM standard. But let's do this on another PR, here the focus is the split.